timestream.lua

```diff
@@ -0,0 +1,339 @@
+-- speeds up the calendar, units, or both
+
+--[====[
+
+timestream
+==========
+This is only for fortress mode.
```

Comment:
I added a check that would make sure it will only run once in fortress mode. I don't want to give an error because I'd like to have it run by default, unless there's a more elegant way of doing it.

Reply:
Ok, I think that makes sense